### PR TITLE
Updated Swahili README

### PR DESCRIPTION
**Pull request by fvaysh**
Issue Number - #3017 

I have taken time to process each part one by one and modified and improved it as per the conditions you mentioned.
Please check it and merge it.

In this commit, I have significantly improved the clarity and comprehensibility of the Swahili instructions. 
I made sure that the instructions are crystal clear. Users should now find it easier to understand and follow the guidelines provided.
I making improvements, we carefully maintained the original style and tone of the instructions. This ensures that the code has not been changed too much and it matched it with your original English ReadME file.
I fixed typos and grammar issues to ensure the instructions are error-free.

### CODE REVIEW

**Comment by CandidDeer** [comment]
@Syknapse Please let me know if we can merge this. Thx.

**Comment by fvaysh** [comment]
> @Syknapse Please let me know if we can merge this. Thx.

Can I please have the update on this, I already have checked for cases and improved it properly. So, can you please see if it can be merged?

**Comment by fvaysh** [comment]
@Syknapse heya, sorry to disturb, but can you merge this if everything works fine, I fixed up typos, improved clarity, and took me time for this, can you please check and merge it ?

File: translations/README/SWAHILI.md [hunk 1]
```diff
@@ -7,8 +7,8 @@
 [![Tweet](https://img.shields.io/twitter/url/http/shields.io.svg?style=social)][twit]
 
 [![Discord](https://badgen.net/discord/online-members/tWkvS4ueVF?label=Jiunge%20Our%20Discord%20Server&icon=discord)](https://discord.gg/tWkvS4ueVF 'Jiunge na seva yetu ya Discord !')
-[![PRs Karibu](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](https://syknapse.github.io/Contribute-To-This -Mradi/)
-[![Open Source Love](https://badges.frapsoft.com/os/v2/open-source.svg?v=103)](https://syknapse.github.io/Changia-Kwa-Hii- Mradi/)
+[![PRs Karibu](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](https://syknapse.github.io/Contribute-To-This-Mradi/)
+[![Open Source Love](https://badges.frapsoft.com/os/v2/open-source.svg?v=103)](https://syknapse.github.io/Changia-Kwa-Hii-Mradi/)
```
Comment (TenTraicion) [inline translations/README/SWAHILI.md:11]: the links shouldn't be translated. kindly reverse this and change it to same as original README file (english)

File: translations/README/SWAHILI.md [hunk 2]
```diff
@@ -337,12 +337,10 @@ Ninaahidi nitajaribu kukagua na kuunganisha haraka iwezekanavyo lakini ninafanya
...
 - Ili kusaidia kuboresha jumuiya yetu, angalia kichupo cha GitHub [Majadiliano](https://github.com/Syknapse/Contribute-To-This-Project/discussions) kilicho karibu na Vuta Maombi. Eneo hili ni mahali pa kujitambulisha, kuingia katika majadiliano ya kina kuhusu Chanzo Huria, na kuwasiliana na Wasimamizi wa Mradi. Je, utatusaidia kujenganje ya kipengele hiki na kuboresha jumuiya yetu?
-- **Asante kwa kuchangia mradi huu**. Sasa unaweza kuendelea na kujaribu kuchangia miradi mingine; tafuta ![Toleo zuri la Kwanza](https://user-images.githubusercontent.com/29199184/33852733-e23b7070-debb-11e7-907b-4e7a03aad436.png) lebo kwa chaguo za michango zinazofaa kwa wanaoanza.
+- **Asante kwa kuchangia mradi huu**. Sasa unaweza kuendelea na kujaribu kuchangia miradi mingine; tafuta ![Toleo zuri la Kwanza](https://user-images.githubusercontent.com/29199184/33852733-e23b70 70-debb-11e7-907b-4e7a03aad436.png) lebo kwa chaguo za michango zinazofaa kwa wanaoanza.
```
Comment (TenTraicion) [inline translations/README/SWAHILI.md:340]: again the links shoudn't be changed in any circumstances. they should be same as original README (eng)

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@fvaysh Thanks for your contribution, k9indly make those changes, and mention me after the changes.

**Pull request closed without merging**